src/ec/packages/ec-component-hero-banner/hero-banner.story.js

```diff
@@ -0,0 +1,104 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import bannerDataDefault from '@ecl/ec-specs-hero-banner/demo/data--default';
+import bannerDataImage from '@ecl/ec-specs-hero-banner/demo/data--image';
+import bannerDataImageShade from '@ecl/ec-specs-hero-banner/demo/data--image-shade';
+import bannerDataPrimary from '@ecl/ec-specs-hero-banner/demo/data--primary';
+import bannerDataAlignLeft from '@ecl/ec-specs-hero-banner/demo/data--align-left';
+
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+
+import heroBanner from './hero-banner.html.twig';
+import heroBannerDocs from './docs/hero-banner.md';
+
+function formatBanner(b) {
+  const iconType = b.button.icon.shape.split('--');
+  const banner = {
+    type: b.variant,
+    title: text('Title', b.title),
+    description: text('Description', b.description),
+    button: {
+      type: b.button.variant,
+      label: text('Button label', b.button.label),
+      icon: {
+        type: iconType[0],
+        name: iconType[1],
+        transform: b.button.icon.transform,
+        size: b.button.icon.size,
+        path: defaultSprite,
+      },
+    },
+    centered: boolean('Centered', b.isCentered),
+  };
+
+  if ('image' in b) {
+    banner.image = 'https://v2--europa-component-library.netlify.com'.concat(
+      b.image
+    );
+  }
+
+  return banner;
+}
+
+storiesOf('Components/HeroBanner', module)
```

Comment:
please add a space between Hero and Banner

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/49/commits/b61ae00e3ae1e87c10ce6b1bddf1950e0fae7785